plugins/org.jkiss.dbeaver.registry/OSGI-INF/l10n/bundle.properties

```
@@ -28,6 +28,7 @@ language.ja.label=Japanese
 language.uk.label=Ukrainian
 language.ko.label=Korean
 language.uk.label=Ukrainian
+language.ro_RO.label=Romanian
```

This should be
```suggestion
language.ro.label=Romanian
```

Additionally, this won't do anything. In order to be able to change the language from DBeaver, you need to register Romanian in a list of supported languages here:
https://github.com/dbeaver/dbeaver/blob/93b491375bee5996407e5fa412b04562730f365b/plugins/org.jkiss.dbeaver.registry/plugin.xml#L25-L38

Add the following line at the end:
```diff
         <language code="es" label="%language.es.label"/>
         <language code="tw" label="%language.tw.label"/>
         <language code="uk" label="%language.uk.label"/>
+        <language code="ro" label="%language.ro.label"/>
     </extension>
```